components/molecules/AvailableResources.js

```diff
@@ -0,0 +1,26 @@
+import ResourceCard from "../atoms/ResourceCard";
+import PropTypes from "prop-types";
+
+/**
+ *  Available Resources Component
+ */
+export default function AvailableResources(props) {
+  return (
+    <div className="pt-3">
+      <h4>{props.title}</h4>
```

Comment:
```suggestion
      <p className="text-h4 font-display font-bold">{props.title}</p>
```
axe tool complains about having headings only increase by one, so style this like an h4 rather than have it as an h4. The above code styles the text like the Figma designs